api/src/test/scala/hmda/api/http/LarHttpApiSpec.scala

```diff
@@ -97,22 +97,31 @@ class LarHttpApiSpec extends WordSpec with MustMatchers with ScalatestRouteTest
     }
   }
 
-  /*
-  "reject requests without 'CFPB-HMDA-Username' header" in {
-    // Request the endpoint without username header (but with other headers)
-    Post("/lar/parse", larCsv).addHeader(institutionsHeader) ~> larRoutes ~> check {
-      status mustBe StatusCodes.FORBIDDEN
-      responseAs[ErrorResponse] mustBe ErrorResponse(403, "Unauthorized Access", "")
+  "LAR API Authorization and rejection handling" must {
+    // These endpoints are not protected by authorization, so they should have
+    // the same result whether or not the request contains auth headers.
+
+    "allow requests without 'CFPB-HMDA-Institutions' header" in {
+      Post("/lar/parse", larCsv).addHeader(usernameHeader) ~> larRoutes ~> check {
+        status mustEqual StatusCodes.OK
+        responseAs[LoanApplicationRegister] mustBe lar
+      }
     }
-  }
 
-  "reject requests without 'CFPB-HMDA-Institutions' header" in {
-    // Request the endpoint without institutions header (but with other headers)
-    Post("/lar/parse", larCsv).addHeader(usernameHeader) ~> larRoutes ~> check {
-      status mustBe StatusCodes.FORBIDDEN
-      responseAs[ErrorResponse] mustBe ErrorResponse(403, "Unauthorized Access", "")
+    "allow requests without 'CFPB-HMDA-Username' header" in {
+      Post("/lar/validate", lar).addHeader(institutionsHeader) ~> larRoutes ~> check {
+        status mustEqual StatusCodes.OK
+        responseAs[List[ValidationError]] mustBe Nil
+      }
+    }
+
+    "not handle routes that aren't defined in this API" in {
+      // Request the endpoint without username header (but with other headers)
```

Comment:
This comment seems to conflict with the line below.  Doesn't `getWithCfpbHeaders()` add the `CFPB-HMDA-Username` header?  I may also be confused as I'm just basing what `getWithCfpbHeaders()` does other tests.